lib/options.ts

```diff
@@ -38,10 +38,11 @@ export class Options extends commonOptionsLibPath.OptionsBase {
 			bundle: { type: OptionType.Boolean },
 			all: { type: OptionType.Boolean },
 			teamId: { type: OptionType.String },
-			rebuild: { type: OptionType.Boolean, default: true },
 			syncAllFiles: { type: OptionType.Boolean },
 			liveEdit: { type: OptionType.Boolean },
-			chrome: { type: OptionType.Boolean }
+			chrome: { type: OptionType.Boolean },
+			clean: { type: OptionType.Boolean },
+			watch: { type: OptionType.Boolean, default: true },
```

Comment:
Technically, yes. Someone could argue that this way it is easier to add new entries, however different languages, different styles.. removed the comma.